.github/workflows/build_and_test_linux.yml

```diff
@@ -0,0 +1,101 @@
+name: Build and test Linux
+
+on:
+ push:
+   branches:
+     - main
+   tags: "*"
+ pull_request:
+
+env:
+  UV_FROZEN: true         # https://docs.astral.sh/uv/configuration/environment/#uv_frozen
+
+jobs:
+  build-wheels-linux:
+    name: Build 🛞
+    timeout-minutes: 120
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ '3.11', '3.12' ]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: build-wheels-linux
+        uses: docker://quay.io/pypa/manylinux2014_x86_64
+        env:
+          PYTHON_VERSION: ${{ matrix.python-version }}
+        with:
+          entrypoint: /bin/bash
+          args: '-c "sh wheel_build_scripts/build_wheels_linux.sh ${{ matrix.python-version }}"'
+
+      - name: Upload wheel as artifact
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: Python ${{ matrix.python-version }} wheel
+          path: |
+            graphite*.whl
+
+  test-linux:
```

Comment:
Think we should also make a step for running the actual tests in graphite-maps, and the rather call this integration tests against ERT.